arbitrum-docs/node-running/running-an-archive-node.mdx

```diff
@@ -0,0 +1,61 @@
+# Running an Archive Node
+
+Note: If you’re interested in accessing the Arbitrum network but you don’t want to setup your own node, see our [Node Providers](./node-providers.mdx) to get RPC access to fully-managed nodes hosted by a third party provider.
+
+### Do You Need To Run An Archive Node?
+
+An archival node is a full (nitro) node that also builds an archive of historical states. Note that the use cases for having an archival node are rare (e.g., if you need to process historical data), and the great majority of users don't need it.
+
+- Followings specify the minimum hardware configuration required to setup an Arbitrum archival node:
+  - RAM: 16GB+ for Nitro and 32GB+ for Classic
+  - CPU: 4+ core CPU
+  - Storage: Minimum 2TB SSD (make sure it is extendable)
+  - Estimated Growth Rate: Around 128GB per week
+
+❗️Note: The minimum storage requirements will change over time as the Nitro chain grows. It is recommended to use more than the minimum requirements to run a robust archival node.
+
+### Required Artifacts
+
+- Latest Docker Image: `offchainlabs/nitro-node:v2.0.11-8e786ec`
+
+- Arbitrum One Nitro Genesis Database Snapshot
+
+  - Use the parameter `--init.url="https://snapshot.arbitrum.io/mainnet/nitro.tar"` on first startup to initialize Nitro database
+  - sha256 checksum: `a609773c6103435b8a04d32c63f42bb5fa0dc8fc38a2acee4d2ab2d05880205c`
+  - size: 33.5573504 GB
+
+- Other chains do not have classic blocks, and do not require an initial genesis database
+
+### Required parameter
+
+- `--l1.url=<Layer 1 Ethereum RPC URL>`
+  - Must provide standard layer 1 node RPC endpoint that you run yourself or from a node provider
+- `--l2.chain-id=<L2 Chain ID>`
+  - See [public chains](../public-chains.mdx) for a list of Arbitrum chains and the respective L2 Chain Ids
+- `--node.caching.archive`
+  - Retain past block state
+- `--node.rpc.classic-redirect=<classic node RPC>`
+  - Redirects archive requests for pre-nitro blocks to the designated RPC, which should be an Arbitrum Classic node with archive database. Only valid for Arbitrum One.
+
+### Important ports
+
+- RPC: `8547`
+- Sequencer Feed: `9642`
+- WebSocket: `8548`
+
+### Putting it all together
+
+- When running docker image, an external volume should be mounted to persist the database across restarts. The mount point should be `/home/user/.arbitrum/mainnet`.
+  - Here is an example of how to run an archival node:
+  ```shell
```

Comment:
seems we only provide classic config command here, maybe we also provide nitro one and tell user what is the differences between those two, and what kind node they want to run.